pkg/recipes/driver/bicep.go

```diff
@@ -403,3 +434,28 @@ func (d *bicepDriver) getGCOutputResources(current []string, previous []string)
 
 	return diff, nil
 }
+
+func (d *bicepDriver) FindSecretIDs(ctx context.Context, envConfig recipes.Configuration, definition recipes.EnvironmentDefinition) (secretStoreIDResourceKeys map[string][]string, err error) {
```

Comment:
does this function get called?

Reply:
yes, it is called from recipe `Engine`, we dont have to make any changes in Engine as we are just implementing a DriverWithSecret interface here , it automatically does the part of calling this function